images/base/containerd-config.toml

```diff
@@ -2,10 +2,15 @@
 disabled_plugins = ["aufs", "btrfs", "zfs"]
 
 # set default runtime handler to v2, which has a per-pod shim
-[plugins.cri.containerd.default_runtime]
-  runtime_type="io.containerd.runc.v2"
+[plugins."io.containerd.grpc.v1.cri".containerd]
+  default_runtime_name="runc"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+  runtime_type = "io.containerd.runc.v2"
 
-# Setup a runtime with the magic name ("test-handler") used for Kubernetes
-# runtime class tests ...
+# Setup a runtime with the magic name ("test-handler") for runtime class tests
 [plugins.cri.containerd.runtimes.test-handler]
   runtime_type = "io.containerd.runc.v2"
+
+# import any additional config files written by kind, and any user written
+# config files (/etc/containerd/user-config/*.toml)
+imports = ["/kind/containerd/*.toml", "/etc/containerd/user/*.toml"]
```

Comment:
i think maybe this needs to move up? so far this part isn't working, the rest of the config is definitely being loaded though

/hold

Reply:
testing more pending https://github.com/kubernetes-sigs/kind/pull/954

Reply:
yeah, the examples show it always at the top of the file

Reply:
I did this locally but haven't gotten a registry mirror override working yet.

I need to brush up on toml, but it seems:
- indent is just for humans, not syntax
- any top level keys need to be before any tables

Reply:
pushed the moved up re-do. still need to test more with dropins. working on it... will resume in the morning (have wg-k8s-infra meeting in ~7 hours ...)